services/openshift-service-impl/src/test/java/io/fabric8/launcher/service/openshift/impl/OpenShiftServiceIT.java

```diff
@@ -186,7 +187,9 @@ public void listProjects() {
 
         //then
         assertNotNull(projects);
-        assertThat(projects).extracting(OpenShiftProject::getName).allMatch(s -> s.startsWith(PREFIX_NAME_PROJECT));
+        assertThat(projects).extracting(OpenShiftProject::getName)
+                .describedAs("expecting to match " + PREFIX_NAME_PROJECT + "*")
+                .allMatch(s -> s.startsWith(PREFIX_NAME_PROJECT));
```

Comment:
We have a known issue that this assertion fails if the OS cluster has other projects. Can you fix that?

Reply:
You mean `myproject`? I just added a description here, but sure.

Reply:
Yes, exactly